src/index.js

```diff
@@ -0,0 +1,12 @@
+import Todolist from './modules/dom.js';
+
+const library = new Todolist();
+
+if (localStorage.getItem('collection')) {
+  library.collection = JSON.parse(localStorage.getItem('collection'));
+  library.displayItems();
+}
+const refresh = document.querySelector('.rolling');
+refresh.addEventListener('click', () => {
+  refresh.classList.toggle('active');
+});
```

Comment:
- [ ] You did great building out your todo webapp, :+1: but I think you did not follow the requirement for this milestone. 

>Project requirements
Set up a new project with webpack that is based on the webpack exercise you have already completed.
Create an index.html file and write your HTML markup here. Create an empty To Do List placeholder (`<div> or <ul>` element). The index.html file must be set as a template using the HTML Webpack Plugin.
Create an index.js file and set an array of some simple to do tasks (array of objects). Each task object should contain three keys:
description [string].
completed [bool].
index: [number].
Write a function to iterate over the tasks array and populate an HTML list item element for each task.
On page load render the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task.
Create a style.css and set rules for the To Do List. CSS must be loaded by Webpack Style/CSS Loader. Your list should be a clone of the part of the minimalist project captured in the video below.

- [ ] Looking at this requirement above, You are meant to 
- - Set up webpack, You can follow t[his lesson here ](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/exercises/exercise_set_up_project_with_webpack.md)
- - Create your index.html and add a div where you can append the todo items
- - Create an array of objects containing todo items
- - Then write a function that loops through your items and displays them on the DOM.
- - Then you can call the function on page load,.

- - I'LL SUGGEST YOU GO THROUGH THE PROJECT REQUIREMENT ONE MORE TIME AND IMPLEMENT THE TASKS FOR THIS MILESTONE. :+1: :smile: :+1:

Reply:
i have configure webpack for js i misunderstand that one, about the array i have made a class and in the contructor i made an array with those specifications, is that okey?